src/main/scala/com/typesafe/sbt/osgi/Osgi.scala

```diff
@@ -181,7 +181,9 @@ private object Osgi {
             import _root_.java.nio.file._
             import _root_.scala.collection.JavaConverters._
             val path = tmpArtifactDirectoryPath.toPath
-            Files.walk(path).iterator.asScala.map(f => f.toFile -> path.relativize(f).toString).filterNot { case (_, p) => p == "META-INF/MANIFEST.MF" }.toTraversable
+            Files.walk(path).iterator.asScala.map(f => f.toFile -> path.relativize(f))
+              .collect { case (f, p) if p != (file("META-INF") / "MANIFEST.MF").toPath => (f, p.toString) }
```

Comment:
So the fix is using the `/` operator?

Reply:
This kind of problem I was fixed in pekko/akka spec too:)

Reply:
> So the fix is using the `/` operator?

Yes its part of the sbt API which is just a wrapper over `new File(a, b)`

Reply:
I guess I understand the reason, Because `line.separator` in Windows cannot be represented with `/`.